device/virtual/x86_64-kvm_x86_64-r0/msft_multi_asic_vs/0/port_config.ini

```diff
@@ -1,33 +1,33 @@
-Ethernet52      49,50,51,52       fortyGigE0/52     13          40000
-Ethernet56      57,58,59,60       fortyGigE0/56     14          40000
-Ethernet60      61,62,63,64       fortyGigE0/60     15          40000
-Ethernet64      69,70,71,72       fortyGigE0/64     16          40000
-Ethernet68      65,66,67,68       fortyGigE0/68     17          40000
-Ethernet72      73,74,75,76       fortyGigE0/72     18          40000
-Ethernet76      77,78,79,80       fortyGigE0/76     19          40000
-Ethernet80      109,110,111,112   fortyGigE0/80     20          40000
-Ethernet84      105,106,107,108   fortyGigE0/84     21          40000
-Ethernet88      113,114,115,116   fortyGigE0/88     22          40000
-Ethernet92      117,118,119,120   fortyGigE0/92     23          40000
-Ethernet96      125,126,127,128   fortyGigE0/96     24          40000
-Ethernet100     121,122,123,124   fortyGigE0/100    25          40000
-Ethernet104     81,82,83,84       fortyGigE0/104    26          40000
-Ethernet108     85,86,87,88       fortyGigE0/108    27          40000
-Ethernet112     93,94,95,96       fortyGigE0/112    28          40000
-Ethernet116     89,90,91,92       fortyGigE0/116    29          40000
-Ethernet120     101,102,103,104   fortyGigE0/120    30          40000
-Ethernet124     97,98,99,100      fortyGigE0/124    31          40000
+# name        lanes            alias          index   asic_port_name    role  speed
+Ethernet0     25,26,27,28      Ethernet1/1    0       Eth0-ASIC0        E     40000
+Ethernet4     29,30,31,32      Ethernet1/2    1       Eth1-ASIC0        E     40000
+Ethernet8     33,34,35,36      Ethernet1/3    2       Eth2-ASIC0        E     40000
+Ethernet12    37,38,39,40      Ethernet1/4    3       Eth3-ASIC0        E     40000
+Ethernet16    45,46,47,48      Ethernet1/5    4       Eth4-ASIC0        E     40000
+Ethernet20    41,42,43,44      Ethernet1/6    5       Eth5-ASIC0        E     40000
+Ethernet24    1,2,3,4          Ethernet1/7    6       Eth6-ASIC0        E     40000
+Ethernet28    5,6,7,8          Ethernet1/8    7       Eth7-ASIC0        E     40000
+Ethernet32    13,14,15,16      Ethernet1/9    8       Eth8-ASIC0        E     40000
+Ethernet36    9,10,11,12       Ethernet1/10   9       Eth9-ASIC0        E     40000
+Ethernet40    17,18,19,20      Ethernet1/11   10      Eth10-ASIC0       E     40000
+Ethernet44    21,22,23,24      Ethernet1/12   11      Eth11-ASIC0       E     40000
+Ethernet48    53,54,55,56      Ethernet1/13   12      Eth12-ASIC0       E     40000
+Ethernet52    49,50,51,52      Ethernet1/14   13      Eth13-ASIC0       E     40000
+Ethernet56    57,58,59,60      Ethernet1/15   14      Eth14-ASIC0       E     40000
+Ethernet60    61,62,63,64      Ethernet1/16   15      Eth15-ASIC0       E     40000
+Ethernet64    69,70,71,72      Eth16-ASIC0    0       Eth16-ASIC0       I     40000
+Ethernet68    65,66,67,68      Eth17-ASIC0    1       Eth17-ASIC0       I     40000
```

Comment:
Backplane interface here for ASIC0 ?

Reply:
Fixed the interface names.

Reply:
do we want to have "Ext", "Int", wonder if in the future we will have more role/type? for example, fabric link.

Reply:
Updated as per suggestion.